cmd/internal/cli/singularity.go

```diff
@@ -188,6 +188,22 @@ func setSylogColor() {
 	}
 }
 
+// createDataDir will create ~/.singularity if it does not already exist
+func createDataDir(usr *user.User) {
+	dataDir := path.Join(usr.HomeDir, ".singularity")
```

Comment:
I'll second @gvallee's point that we should use `syfs.ConfigDir()` to generate the directory path. I think it's best to use that consistently throughout the cli